server/src/concepts/user/controllers/index.js

```diff
@@ -13,3 +14,12 @@ export const createUser = async (req, res) => {
   }
   return res.status(200).send(`User with ${req.body.email} was created`);
 };
+
+export const updateUser = async (req, res) => {
+  try {
```

Comment:
Please add validation for updating user. It will be similar to createUser but for now we have id in request

Reply:
I've already added validation in commands and it works.

Reply:
ok, good 👍